extensions/amp-story/1.0/amp-story.js

```diff
@@ -2668,10 +2668,14 @@ export class AmpStory extends AMP.BaseElement {
   fetchLocalizationStrings_(languageCode) {
     const localizationService = getLocalizationService(this.element);
 
-    const localizationUrl = `${calculateScriptBaseUrl(
+    const base = calculateScriptBaseUrl(
       this.win.location,
       getMode(this.win).localDev
-    )}/v0/amp-story.${languageCode}.json`;
+    );
+    const rtvPath = getMode(this.win).localDev
+      ? ''
+      : `rtv/${getMode(this.win).rtvVersion}/`;
+    const localizationUrl = `${base}/${rtvPath}v0/amp-story.${languageCode}.json`;
```

Comment:
Isn't this just `calculateExtensionScriptUrl` or `calculateEntryPointScriptUrl` (note, not `calculateScriptBaseUrl`)

Reply:
yeah it is. the only problem is that `calculateExtensionScriptUrl` and `calculateEntryPointScriptUrl` both assume a javascript extension. should i go ahead and update them so they can take in an optional extension type?

Reply:
Or create a new helper that can accommodate JSON. Just put it in the same file to localize the URL generation.

Reply:
got it. ill make the change